test/ownership/HasNoEther.test.js

```diff
@@ -4,61 +4,60 @@ const { ethSendTransaction, ethGetBalance } = require('../helpers/web3');
 const HasNoEtherTest = artifacts.require('HasNoEtherTest');
 const ForceEther = artifacts.require('ForceEther');
 
-contract('HasNoEther', function (accounts) {
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('HasNoEther', function ([_, owner, anyone]) {
   const amount = web3.toWei('1', 'ether');
 
-  it('should be constructible', async function () {
-    await HasNoEtherTest.new();
+  beforeEach(async function () {
+    this.hasNoEther = await HasNoEtherTest.new({ from: owner });
   });
 
   it('should not accept ether in constructor', async function () {
     await expectThrow(HasNoEtherTest.new({ value: amount }));
   });
 
   it('should not accept ether', async function () {
-    const hasNoEther = await HasNoEtherTest.new();
-
     await expectThrow(
       ethSendTransaction({
-        from: accounts[1],
-        to: hasNoEther.address,
+        from: owner,
+        to: this.hasNoEther.address,
         value: amount,
       }),
     );
   });
 
   it('should allow owner to reclaim ether', async function () {
-    // Create contract
-    const hasNoEther = await HasNoEtherTest.new();
-    const startBalance = await ethGetBalance(hasNoEther.address);
+    const startBalance = await ethGetBalance(this.hasNoEther.address);
     assert.equal(startBalance, 0);
 
     // Force ether into it
     const forceEther = await ForceEther.new({ value: amount });
-    await forceEther.destroyAndSend(hasNoEther.address);
-    const forcedBalance = await ethGetBalance(hasNoEther.address);
+    await forceEther.destroyAndSend(this.hasNoEther.address);
+    const forcedBalance = await ethGetBalance(this.hasNoEther.address);
     assert.equal(forcedBalance, amount);
 
     // Reclaim
-    const ownerStartBalance = await ethGetBalance(accounts[0]);
-    await hasNoEther.reclaimEther();
-    const ownerFinalBalance = await ethGetBalance(accounts[0]);
-    const finalBalance = await ethGetBalance(hasNoEther.address);
+    const ownerStartBalance = await ethGetBalance(owner);
+    await this.hasNoEther.reclaimEther({ from: owner });
+    const ownerFinalBalance = await ethGetBalance(owner);
+    const finalBalance = await ethGetBalance(this.hasNoEther.address);
     assert.equal(finalBalance, 0);
-    assert.isAbove(ownerFinalBalance, ownerStartBalance);
+    ownerFinalBalance.should.be.bignumber.gt(ownerStartBalance);
```

Comment:
I know this doesn't belong here, but the previous test (`assert.isAbove`) was not actually working, since bignumbers are not supported.